### DISCUSSION TITLE
Including other assets from srcdir for AwsCdkConstructLibrary

### DISCUSSION THREAD

**Discussion by pchaganti** [projen/projen#2296] [Q&A] [answered]
Trying to package a construct using AwsCdkConstructLibrary. The construct contains some additions resources it uses in an assets folder under src. When I publish it as a package, looks like the assets folder is not being included. Is there a setting I am missing or another way to do this?

thanks!

**Comment by mrgrain** [comment, marked as answer]
`src` is excluded from packaging. The easiest way is to locate non compiled assets in a separate folder in the root directory.

**Comment by pchaganti** [comment]
Will try that. Thanks!

**Comment by pchaganti** [comment]
This worked fine. Thanks for your help!
